models/brats_mri_axial_slices_generative_diffusion/docs/README.md

```diff
@@ -0,0 +1,161 @@
+
+For more details usage instructions, visit the [MONAI Bundle Configuration Page](https://docs.monai.io/en/latest/config_syntax.html).
+
+#### Execute Autoencoder Training (w/data download)
+
+Assuming the current directory is the bundle directory, the following command will train the autoencoder network for 1500 epochs using the BraTS dataset. If the dataset is not downloaded, it will be automatically downloaded and extracted to `./Task01_BrainTumour`.
+
+```
+python -m monai.bundle run --config_file configs/train_autoencoder.json --dataset_dir ./ --download_brats True
+```
+
+#### Execute Autoencoder Training
+If the dataset is already downloaded, make sure that `"dataset_dir"` in `configs/train_autoencoder.json` has the correct path to the dataset `Task01_BrainTumour`. Then, run:
+
+```
+python -m monai.bundle run --config_file configs/train_autoencoder.json
+```
+
+#### Override the `train` config to execute multi-GPU training for Autoencoder:
+To train with multiple GPUs, use the following command, which requires scaling up the learning rate according to the number of GPUs. Keep in mind that this command will take approximately 9 hours to complete when using 8 GPUs, each with 32G of memory.
+
+```
+torchrun --standalone --nnodes=1 --nproc_per_node=8 -m monai.bundle run --config_file "['configs/train_autoencoder.json','configs/multi_gpu_train_autoencoder.json']" --lr 2e-4
+```
+
+#### Execute Latent Diffusion Model Training
+After training the autoencoder, run the following command to train the latent diffusion model. This command will print out the scale factor of the latent feature space. If your autoencoder is well trained, this value should be close to 1.0.
+
+```
+python -m monai.bundle run --config_file "['configs/train_autoencoder.json','configs/train_diffusion.json']"
+```
+
+#### Override the `train` config to execute multi-GPU training for Latent Diffusion Model:
+To train with multiple GPUs, use the following command, which requires scaling up the learning rate according to the number of GPUs. Keep in mind that this command will take approximately 5 hours to complete when using 8 GPUs, each with 32G of memory.
+```
+torchrun --standalone --nnodes=1 --nproc_per_node=8 -m monai.bundle run --config_file "['configs/train_autoencoder.json','configs/train_diffusion.json','configs/multi_gpu_train_autoencoder.json','configs/multi_gpu_train_diffusion.json']"  --lr 8e-5
+```
+
+
+#### Execute inference:
+The following code generates a synthetic image from a random sampled noise.
+```
+python -m monai.bundle run --config_file configs/inference.json
+```
+The generated image will be saved to `./output`
+
+![Example synthetic image](https://developer.download.nvidia.com/assets/Clara/Images/monai_brain_image_gen_ldm2d_example_generation.png)
+
```

Comment:
Are we missing the TorchScript export or does 2D not having one?

Reply:
It is not supported. I submitted issue:
https://github.com/Project-MONAI/GenerativeModels/issues/387
https://github.com/Project-MONAI/GenerativeModels/issues/288